lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -93,6 +93,10 @@
     choices: [ "present", "started", "stopped", "absent", "drained" ]
     required: False
     default: "started"
+  ignore_time_wait:
+    description:
+      - ignore time_wait socket states when waiting for connections to be drained
+    required: false
```

Comment:
Options added after a module has been released need version data. Please add
`version_added: "2.3"` in this section